lib/portal/dependencies.properties

```diff
@@ -6,12 +6,15 @@ hibernate-core=org.hibernate:hibernate-core:5.6.7.Final
 hikaricp=com.zaxxer:HikariCP:2.6.3
 istack-commons-runtime=com.sun.istack:istack-commons-runtime:3.0.7
 jabsorb=com.liferay:org.jabsorb:1.3.2.LIFERAY-PATCHED-1
+jackson-databind=com.fasterxml.jackson.core:jackson-databind:2.13.2.2
 javassist=org.javassist:javassist:3.18.2-GA
 jaxb-api=javax.xml.bind:jaxb-api:2.3.0
 jaxb-runtime=org.glassfish.jaxb:jaxb-runtime:2.3.1
 jaxen=jaxen:jaxen:1.1.6
+jaxrs-api=javax.ws.rs:javax.ws.rs-api:2.1
```

Comment:
@javalosjr96 please clean up the history to avoid adding random changes and remove them later.

Reply:
@shuyangzhou 
I'm not really sure where this change came from, can you fetch the latest master so that the comparison could be more accurate with the changes being made in this PR?

Reply:
NVM